espnet/nets/pytorch_backend/e2e_asr_mix_transformer.py

```diff
@@ -0,0 +1,530 @@
+# Copyright 2019 Shigeki Karita
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+"""Transformer speech recognition model (pytorch)."""
```

Comment:
If you can mention your ICASSP2020 paper (https://arxiv.org/pdf/2002.03921.pdf) in this docstring, RESULTS.md (egs), and anywhere, that would be super cool.

Reply:
Done. Thanks!